manager/allocator/allocator_test.go

```diff
@@ -796,6 +796,149 @@ func TestAllocatorRestoreForDuplicateIPs(t *testing.T) {
 	}
 }
 
+// TestAllocatorRestartNoEndpointSpec covers the leader election case when the service Spec
+// does not contains the EndpointSpec.
```

Comment:
nit: does not contains => does not contain

Reply:
done